packages/main-api/src/controllers/m1/assembly_members.rs

```diff
@@ -48,8 +44,31 @@ impl AssemblyMemberControllerV1 {
     ) -> Result<Json<AssemblyMember>> {
         let log = ctrl.log.new(o!("api" => "create_assembly_member"));
         slog::debug!(log, "act_assembly_member {:?}", body);
-        // TODO: implement it
+        let cli = easy_dynamodb::get_client(&log);
+
+        if body == ActionAssemblyMemberRequest::FetchMembers {
+            if let Some(rows) = get_active_members().await?["row"].as_array() {
+                for row in rows {
+                    // korean info
+                    let member: Member = serde_json::from_value(row.clone())?;
+
+                    // profile image
+                    let image_url = get_member_profile_image(member.code.clone()).await?;
+
+                    cli.create(
+                        &AssemblyMember::try_from((member.code.clone(), image_url.clone(), "ko", &member))
+                    )
```

Comment:
```suggestion
            let members =  get_active_members().await?; 
            let image_url = get_member_profile_image(member.code.clone()).await?;
            let doc:AssemblyMember = members.try_into()?.with_profile_url(image_url);
            cli.create(&doc)
```

이런식의 코드가 나올 수 있게 구성되어야 하는게 맞는거 같아요.